corehq/apps/hqadmin/views/users.py

```diff
@@ -398,7 +399,7 @@ def form_valid(self, form):
 
         reset_password = form.cleaned_data['reset_password']
         if reset_password:
-            self.user.set_password(uuid.uuid4().hex)
+            unset_password(self.user)
```

Comment:
The language here seems a little confusing. 'Reset password', to me, means that the password is being changed with the intent of delivering that password to the user. It's likely not a huge deal, but I feel like the most appropriate fix is probably changing the language on this form to 'clear password' or something?

Reply:
I think I'm going to leave that for whenever the next time we're working on this particular form is, since this was peripheral to the main change I was trying to make